hypothesis-python/tests/cover/test_lookup.py

```diff
@@ -810,3 +816,27 @@ def __init__(self, foo: bool = None):
 def test_from_type_can_be_default_or_annotation():
     find_any(st.from_type(AnnotatedAndDefault), lambda x: x.foo is None)
     find_any(st.from_type(AnnotatedAndDefault), lambda x: isinstance(x.foo, bool))
+
+
+@pytest.mark.parametrize("t", BUILTIN_TYPES, ids=lambda t: t.__name__)
+def test_resolves_builtin_types(t):
+    v = st.from_type(t).example()
+    assert isinstance(v, t)
+
+
+@pytest.mark.skipif(sys.version_info[:2] == (3, 6), reason="no ForwardRef type")
```

Comment:
I think that `compat` hack now handles `3.6`, doesn't it?

Reply:
Alas, in 3.6 `_ForwardRef` requires that it be called on a `type` object.  For some reason.

See https://github.com/HypothesisWorks/hypothesis/runs/2084980210?check_suite_focus=true for all the failures when I tried 😭